ports/renesas-ra/pendsv.c

```diff
@@ -132,27 +104,6 @@ __attribute__((naked)) void PendSV_Handler(void) {
         ".no_dispatch:\n"
         #endif
 
-        // Check if there is an active object to throw via nlr_jump
-        "ldr r1, pendsv_object_ptr\n"
-        "ldr r0, [r1]\n"
-        "cmp r0, #0\n"
-        "beq .no_obj\n"
-        #if defined(PENDSV_DEBUG)
```

Comment:
Note the comment just above this, about the debug layout has been invalid and now outdated. Glad to see the `PENDSV_DEBUG` removed, it should be removed from all ports.

Reply:
It is now removed from all ports (although I see there's still a few stray `-DPENDSV_DEBUG` remaining in Makefile's that should be removed).

As for the comments for the stack layout: feel free to open a PR to fix those.

Reply:
> As for the comments for the stack layout: feel free to open a PR to fix those.
I think we should remove, it was never correct; the stack layout doesn't change with different builds, the registers are stacked *by the hardware* so the layout is always the same. I did some extensive testing here:

https://github.com/micropython/micropython/issues/315#issuecomment-577692954